packages/client/src/pages/Game/types.ts

```diff
@@ -0,0 +1,16 @@
+export type TDraw = {
+  space: (params: { canvasContext: CanvasRenderingContext2D }) => void
+  enemies: (params: { canvasContext: CanvasRenderingContext2D }) => void
+  hole: (params: { canvasContext: CanvasRenderingContext2D }) => void
+}
+
+type TObjectState = {
```

Comment:
По названию не очень понятно, я бы называл например entity
```suggestion
type TGameEntity = {
```

Reply:
Спасибо, смотрю сейчас свежим взглядом и имя действительно неудачное